index.ts

```diff
@@ -86,6 +88,32 @@ async function main() {
             }
             continue;
           }
+          // If devpool issue has the "Unavailable" label then close such issue
+          if ((devpoolIssue.labels as GitHubLabel[]).some((label) => label.name.includes(LABELS.UNAVAILABLE))) {
```

Comment:
Shouldn't we check the availability of the project issue here instead of the "unavailable" label? So if project issue is assigned then force close the devpool issue.